builtin/logical/pki/path_resign_crls.go

```diff
@@ -0,0 +1,274 @@
+package pki
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/base64"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/certutil"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const (
+	crlNumberParam      = "crl_number"
+	deltaCrlNumberParam = "delta_crl_number"
```

Comment:
Maybe we want to call this `delta_crl_base_ref_number` or similar? Or without the `_ref`? Just to make it more clear what is what.

Reply:
Done.